package.json

```diff
@@ -18,12 +18,13 @@
     "lint"
   ],
   "dependencies": {
-    "standard-http-error": "2.0.0"
+    "@types/standard-http-error": "^2.0.1",
+    "standard-http-error": "2.0.1"
   },
   "devDependencies": {
     "@uphold/github-changelog-generator": "^0.7.0",
-    "eslint": "^5.0.1",
-    "eslint-config-uphold": "^0.1.1",
+    "eslint": "^8.20.0",
+    "eslint-config-uphold": "^4.1.0",
```

Comment:
These bumps are causing issues on my end, namely `prettier not found` error.

Reply:
This version installs the `prettier` plugin ([src](https://github.com/uphold/eslint-config-uphold/blob/master/package.json#L36)), so not sure what's happening...

Reply:
It installs the plugin, but not prettier itself. It's a [peer dependency](https://github.com/uphold/eslint-config-uphold/blob/master/package.json#L53).

Reply:
Move these bumps to a dedicated commit. If they are causing issues, move them to a new pull request to unblock this one?

Reply:
Reverted these versions